### PR TITLE
feat(card): add Card component for react-native

### PR DESCRIPTION
**Pull request by deorukhkarchaitanya**

### CODE REVIEW

**Comment by deorukhkarchaitanya** [comment]
This PR is ready. We only need to change the shadows for iOS and Android. Waiting for the design team to give me these shadows.

File: src/atoms/Card/Card.native.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  shadow-offset: 0px 4px;
+  shadow-opacity: 1;
+  shadow-radius: 20px;
+  elevation: 20;
+  background-color: ${styles.backgroundColor};
+`;
+
+const Card = ({ children, testID }) => {
+  return (
+    <Space padding={styles.padding()}>
```
Comment (kamleshchandnani) [inline src/atoms/Card/Card.native.js]: things which doesn't have conditions let's put them inline. `padding=[1.5, 1.5, 1.5, 1.5]`

File: src/atoms/Card/__tests__/Card.native.test.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { renderWithTheme } from '../../../_helpers/testing';
+import Text from '../../Text';
+import Card from '../index';
+
+describe('Renders <Card /> correctly', () => {
+  it('should create an empty card with specified props', () => {
+    const { container } = renderWithTheme(
+      <Card
+        height="50%"
```
Comment (kamleshchandnani) [inline src/atoms/Card/__tests__/Card.native.test.js]: Card doesn't accepts all these props 🙈 Did you forget to wrap it in size?

File: src/atoms/Card/__tests__/Card.native.test.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        width="100px"
+        maxHeight="200px"
+        maxWidth="100px"
+        minHeight="200px"
+        minWidth="50px"
+      />,
+    );
+    expect(container).toMatchSnapshot();
+  });
+  it('should create a card with text', () => {
```
Comment (kamleshchandnani) [inline src/atoms/Card/__tests__/Card.native.test.js]: `should render a card`?

File: src/atoms/Card/__tests__/Card.native.test.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        width="100px"
+        maxHeight="200px"
+        maxWidth="100px"
+        minHeight="200px"
+        minWidth="50px"
+      >
+        <Text>{displayText}</Text>
+      </Card>,
+    );
+    const helpTextComponent = getByText(displayText);
```
Comment (kamleshchandnani) [inline src/atoms/Card/__tests__/Card.native.test.js]: `helpText`?

File: src/atoms/Size/Size.native.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```
Comment (kamleshchandnani) [inline src/atoms/Size/Size.native.js]: Rebase with master as size is already there

**Comment by deorukhkarchaitanya** [comment]
I had a different implementation of card earlier which was changed after discussion with the team. Forgot to remove bits of the old implementation :(

File: src/atoms/Card/Card.native.js [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+      <StyledCard {...automation(testID)} variant={variant}>
+        {children}
+      </StyledCard>
+    </Space>
+  );
+};
+
+Card.propTypes = {
+  children: PropTypes.node,
+  variant: PropTypes.oneOf(['shadow', 'outline']),
```
Comment (kamleshchandnani) [inline src/atoms/Card/Card.native.js]:
```suggestion
  variant: PropTypes.oneOf(['shadowed', 'outlined']),
```

File: src/atoms/Card/Card.web.js [hunk 7]
```diff
@@ -0,0 +1,3 @@
+const Card = () => null;
+
+export default Card;
```
Comment (kamleshchandnani) [inline src/atoms/Card/Card.web.js]:
```suggestion
const Card = () => <div<WIP</div>;

export default Card;
```

**Pull request merged**